<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReplaceFolioWidget</class>
 <widget class="QWidget" name="ReplaceFolioWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <property name="usesScrollButtons">
      <bool>true</bool>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Principales</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="9" column="1">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Indice Rev</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Localisation</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Fichier :</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLineEdit" name="m_title_le">
           <property name="toolTip">
            <string>Disponible en tant que %title pour les modèles de cartouches</string>
           </property>
           <property name="placeholderText">
            <string>Ne pas modifier</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLineEdit" name="m_author_le">
           <property name="toolTip">
            <string>Disponible en tant que %author pour les modèles de cartouches</string>
           </property>
           <property name="placeholderText">
            <string>Ne pas modifier</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Auteur :</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Date :</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Installation :</string>
           </property>
          </widget>
         </item>
         <item row="9" column="3">
          <widget class="QLineEdit" name="m_indice">
           <property name="toolTip">
            <string>Disponible en tant que %indexrev pour les modèles de cartouches</string>
           </property>
           <property name="placeholderText">
            <string>Ne pas modifier</string>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QLineEdit" name="m_file_le">
           <property name="toolTip">
            <string>Disponible en tant que %filename pour les modèles de cartouches</string>
           </property>
           <property name="placeholderText">
            <string>Ne pas modifier</string>
           </property>
          </widget>
         </item>
         <item row="10" column="3">
          <layout class="QGridLayout" name="gridLayout"/>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Folio :</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0,0">
           <item row="1" column="0">
            <widget class="QRadioButton" name="m_no_date_rb">
             <property name="text">
              <string>Pas de date</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QDateEdit" name="m_date_edit">
             <property name="toolTip">
              <string>Disponible en tant que %date pour les modèles de cartouches</string>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QRadioButton" name="m_fixed_date_rb">
             <property name="text">
              <string>Date fixe :</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="m_date_now_pb">
             <property name="toolTip">
              <string>Appliquer la date actuelle</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../../../qelectrotech.qrc">
               <normaloff>:/ico/22x22/start.png</normaloff>:/ico/22x22/start.png</iconset>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QRadioButton" name="m_unchanged_date">
             <property name="text">
              <string>Ne pas modifier</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="5" column="3">
          <widget class="QLineEdit" name="m_folio_le">
           <property name="toolTip">
            <string>Disponible en tant que %folio pour les modèles de cartouches
Les variables suivantes sont utilisables :
- %id : numéro du folio courant dans le projet 
- %total : nombre total de folios dans le projet 
- %autonum : Folio Auto Numeration</string>
           </property>
           <property name="placeholderText">
            <string>Ne pas modifier</string>
           </property>
          </widget>
         </item>
         <item row="8" column="3">
          <widget class="QLineEdit" name="m_loc">
           <property name="toolTip">
            <string>Disponible en tant que %locmach pour les modèles de cartouches</string>
           </property>
           <property name="placeholderText">
            <string>Ne pas modifier</string>
           </property>
          </widget>
         </item>
         <item row="6" column="3">
          <widget class="QLineEdit" name="m_mach">
           <property name="toolTip">
            <string>Disponible en tant que %machine pour les modèles de cartouches</string>
           </property>
           <property name="placeholderText">
            <string>Ne pas modifier</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Titre :</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QCheckBox" name="m_title_cb">
           <property name="toolTip">
            <string>Supprimer ce texte</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QCheckBox" name="m_author_cb">
           <property name="toolTip">
            <string>Supprimer ce texte</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="4" column="4">
          <widget class="QCheckBox" name="m_file_cb">
           <property name="toolTip">
            <string>Supprimer ce texte</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="5" column="4">
          <widget class="QCheckBox" name="m_folio_cb">
           <property name="toolTip">
            <string>Supprimer ce texte</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="6" column="4">
          <widget class="QCheckBox" name="m_mach_cb">
           <property name="toolTip">
            <string>Supprimer ce texte</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="8" column="4">
          <widget class="QCheckBox" name="m_loc_cb">
           <property name="toolTip">
            <string>Supprimer ce texte</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="9" column="4">
          <widget class="QCheckBox" name="m_indice_cb">
           <property name="toolTip">
            <string>Supprimer ce texte</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Personnalisées</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QVBoxLayout" name="m_tab2_vlayout">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Vous pouvez définir ici vos propres associations noms/valeurs pour que le cartouche en tienne compte. Exemple :
associer le nom &quot;volta&quot; et la valeur &quot;1745&quot; remplacera %{volta} par 1745 dans le cartouche.</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../qelectrotech.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_fixed_date_rb</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_date_edit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_fixed_date_rb</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_date_now_pb</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
